lib/ansible/plugins/inventory/openstack.py

```diff
@@ -306,6 +306,10 @@ def _append_hostvars(self, hostvars, groups, current_host,
             openstack=server)
         self.inventory.add_host(current_host)
 
+        metadata = server.get('metadata', {})
+        for meta_key in metadata:
```

Comment:
```suggestion
        for meta_key, meta_value in server.get('metadata', {}):
            hostvars[current_host][meta_key] = meta_value
```

Reply:
I think you should also remove line 312. Am I wrong?